### ISSUE TITLE
Bug in fourier_transform

### ISSUE THREAD

**Issue by mathcube7** [sympy/sympy#22787]
both the functions `fourier_transform` and `_fourier_transform` give the (same but) wrong result, when applied to the function `sin(x)**2/x**2`. Here is the sample code:

```python
from sympy.abc import *
from sympy import *
from sympy.integrals.transforms import _fourier_transform, fourier_transform

f = sin(x)**2/x**2
F = _fourier_transform(f, x, k, a=1/(2*pi), b=-1, name='F')
plot(F, (k, -3, 3))
```

![image](https://user-images.githubusercontent.com/95348725/147909830-9a605453-597c-4e03-b4b6-f15ad1142d99.png)

or in formula

![image](https://user-images.githubusercontent.com/95348725/147909854-c3850b65-0719-46b1-80da-9c7f118feb0b.png)

This cannot be true, however. As `f` is real, the Fourier transform must satisfy F*(k) = F(-k). And since the Fourier transform itself is real, it must be even: F(k)=F(-k). Obviously, the result of sympy is not.

Sympy version used: 1.9

**Comment by theartpiece** [comment]
can I be assigned @oscarbenjamin

**Comment by anutosh491** [comment]
@theartpiece issues are not generally assigned in sympy . If you want to try it out surely you could go forward and if you face any difficulties during debugging or don't understand something you can comment that here .

**Comment by theartpiece** [comment]
@mathcube7 in general this function seems to behave incorrectly.
Like Try f=e^(-|t|)

**Comment by theartpiece** [comment]
yeah so basically the problem lies here https://github.com/sympy/sympy/blob/00d6469eafdd4aac346a0b598184c15f2560dbe5/sympy/integrals/transforms.py#L1499.
First of all, the problem occurs only when the transformation returns a piecewise function. In this case, fourier returns the first part of the piecewise function (where I think the author assumed that the first part corresponds to the positive case, i.e., x>0 ). 
SUGGESION- Add another function f in the return output such that f(x)=f(-x).

**Comment by theartpiece** [comment]
@oscarbenjamin Let me know if it makes sense and I'll make a PR.

**Comment by oscarbenjamin** [comment]
That code definitely looks suspect. In this case there are nested Piecewise so `piecewise_fold` should probably be used. Clearly also the conditions of the Piecewise should be checked in some way before simply choosing the first case. Probably the author expected that the condition would be on symbolic parameters rather than the Fourier variable but that should be checked rather than assumed.

**Comment by theartpiece** [comment]
How do you add unit tests?

**Comment by mostlyaman** [comment]
> How do you add unit tests?

Tests are added in the corresponding test file for the file in which the change was made. for integrals/transforms.py, the tests are added in integrals/tests/test_transforms.py 
add a function referencing the issue number which led to the change and add the the tests in that function.
Take this for an example
https://github.com/sympy/sympy/blob/1409fcf748d8d73014cca509f2816acb82af7fc0/sympy/integrals/tests/test_transforms.py#L1032-L1033

I dont think there is a guide for adding unit tests in the Development Workflow. Should I add that to the Development Workflow wiki?

**Comment by oscargus** [comment]
There are:
https://github.com/sympy/sympy/wiki/Writing-tests
https://github.com/sympy/sympy/wiki/Writing-test-cases-in-SymPy

(The problem is probably more that there are 592 wiki-pages...)

**Comment by oscarbenjamin** [comment]
> (The problem is probably more that there are 592 wiki-pages...)

Maybe the dev workflow could link to the other pages. Probably a lot of stuff could be stripped out of the dev workflow and just replaced with "here's a link for more information if needed".

**Comment by mostlyaman** [comment]
> Probably a lot of stuff could be stripped out of the dev workflow and just replaced with "here's a link for more information if needed".

Yes, the dev workflow contains too much content and it may be difficult for a new contributor to use. And stuff like Writing tests and Adding someone to AUTHORS should be a part of Dev workflow as  a new contributor would need to do these things. The wiki page has got too long to easily navigate too.
